modules/nw-egress-ips-object.adoc

```diff
@@ -2,11 +2,24 @@
 //
 // * networking/ovn_kubernetes_network_provider/assigning-egress-ips-ovn.adoc
 
+:_mod-docs-content-type: REFERENCE
 [id="nw-egress-ips-object_{context}"]
 = EgressIP object
 
 The following YAML describes the API for the `EgressIP` object. The scope of the object is cluster-wide; it is not created in a namespace.
 
+////
+[IMPORTANT]
+====
+Specifying `Local` for the `externalTrafficPolicy` parameter is not supported. If you set `Local` for this parameter, service ingress traffic that needs to reach pods gets incorrectly rerouted to the egress node that hosts the EgressIP. This situation causes connections to drop and the service to become unavailable.
+====
+////
+
+[IMPORTANT]
+====
+If a pod uses an `EgressIP` object and a `Service` object, such as `NodePort`, that has the `externalTrafficPolicy` parameter set to `Local` exists in the same namespace, service ingress traffic that needs to reach pods gets incorrectly rerouted to the egress node that hosts the EgressIP. Red{nbsp}Hat does not support this configuration because the configuration causes connections to drop and the service to become unavailable.
```

Comment:
I suggest update to below which is from https://github.com/openshift/openshift-docs/pull/76578/files

```
t was unsupported for EgressIP selected pods to also serve as backends for services with `externalTrafficPolicy` set to `Local`. When attempting this configuration, service ingress traffic reaching the pods was incorrectly rerouted to the egress node hosting the EgressIP. This affected how responses to incoming service traffic connections were handled and led to non-functional services when `externalTrafficPolicy` was set to `Local`, as connections were dropped and the service became unavailable
```